specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-30/disk.json

```diff
@@ -1634,6 +1634,10 @@
         "diskAccessId": {
           "type": "string",
           "description": "ARM id of the DiskAccess resource for using private endpoints on disks."
+        },
+        "tier": {
```

Comment:
tier should be supported in update too? also is tier supported in snapshots? Please confirm with Rahul

Reply:
Already added. And sync with Rahul, tier is not supported in snapshots.